### PR TITLE
feat(ct): allow clearing cached build and test transformations

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright-ct-core/src/program.ts [hunk 1]
```diff
@@ -37,4 +41,20 @@ function addDevServerCommand(program: Command) {
...
+  const command = program.command('clear-caches');
+  command.description('clears build and test caches');
+  command.option('-c, --config <file>', `Configuration file. Can be used to specify additional configuration for the output report.`);
+  command.action(async options => {
+    const configFile = options.config;
+    const config = await loadConfig(configFile);
+    if (!config)
+      return;
+    const { templateDir } = await resolveDirs(config.configDir, config.config);
+    await fs.promises.rm(path.join(templateDir, '.cache'), { recursive: true, force: true }).catch(() => {});
```
Comment (dgozman) [inline packages/playwright-ct-core/src/program.ts:54]: Shall we print `Clearing <dir>...` to be polite?

File: packages/playwright-ct-core/src/program.ts [hunk 2]
```diff
@@ -37,4 +41,20 @@ function addDevServerCommand(program: Command) {
...
+function addClearCacheCommand(program: Command) {
+  const command = program.command('clear-caches');
+  command.description('clears build and test caches');
+  command.option('-c, --config <file>', `Configuration file. Can be used to specify additional configuration for the output report.`);
+  command.action(async options => {
+    const configFile = options.config;
+    const config = await loadConfig(configFile);
+    if (!config)
+      return;
+    const { templateDir } = await resolveDirs(config.configDir, config.config);
```
Comment (dgozman) [inline packages/playwright-ct-core/src/program.ts:53]: Use `{ outDir }` instead? It looks into `use.ctCacheDir`.

File: packages/playwright-ct-core/src/program.ts [hunk 3]
```diff
@@ -37,4 +41,20 @@ function addDevServerCommand(program: Command) {
   });
 }
 
+function addClearCacheCommand(program: Command) {
+  const command = program.command('clear-caches');
+  command.description('clears build and test caches');
+  command.option('-c, --config <file>', `Configuration file. Can be used to specify additional configuration for the output report.`);
```
Comment (dgozman) [inline packages/playwright-ct-core/src/program.ts:47]: I believe the description for this option is a bit misleading :)

File: packages/playwright-ct-core/src/program.ts [hunk 4]
```diff
@@ -37,4 +41,20 @@ function addDevServerCommand(program: Command) {
   });
 }
 
+function addClearCacheCommand(program: Command) {
+  const command = program.command('clear-caches');
+  command.description('clears build and test caches');
```
Comment (dgozman) [inline packages/playwright-ct-core/src/program.ts:46]: Add this command to the docs?

**Review by dgozman — APPROVED** [review]

File: packages/playwright-ct-core/src/program.ts (hunk 4 above)
Comment (pavelfeldman) [reply packages/playwright-ct-core/src/program.ts:46]: Let me be slow with it.

**Pull request merged**